### PR TITLE
331/seed dashboard implement classes box

### PR DESCRIPTION
**Pull request by hiaux0**
- feat(seedDashboard)
  - remove "Class price" and add Class name
  - update cliff and vesting
  - update class cap
  - add class sold
  - remove "$" from caps
  - update max input
  - get the correct user class
  - update time left (resp. starts in)
  - remove verified "Verified by Prime Rating"

- fix(seedDashboard)
  - timeLeft, and sorry-not-whitelisted or geoblocked
  - init <timeLeft> when seed is available

- feat(services): add min to BigNumber

- fix(adminDashboard): show classes

- fix(entities): switch order (was showing minus days)

- dev(adminDashboard): add whitelist call
- dev(navbar): add dashboard, and new address

- styles(timeLeft): add width to status with background
- styles(seed.*): make icons round

## TODO
- [ ] confirm max input is working
- [ ] confirm valuation is working
- [ ] adjust exchange ratio placing

### CODE REVIEW

**Comment by hiaux0** [comment]
@giladt 
is becoming a bigger PR, could you start to review this? Ty

File: src/seedSale/seedSale.ts [hunk 1]
```diff
@@ -134,8 +137,23 @@ export class SeedSale {
...
+    // /* prettier-ignore */ console.log(">>>> _ >>>> ~ file: seedSale.ts ~ line 143 ~ this.targetClass.individualCap", this.targetClass.individualCap);
+    // /* prettier-ignore */ console.log(">>>> _ >>>> ~ file: seedSale.ts ~ line 145 ~ this.targetClass.classCap", this.targetClass.classCap);
+    // /* prettier-ignore */ console.log(">>>> _ >>>> ~ file: seedSale.ts ~ line 147 ~ this.seed.userFundingTokenBalance", this.seed.userFundingTokenBalance);
+    // /* prettier-ignore */ console.log(">>>> _ >>>> ~ file: seedSale.ts ~ line 156 ~ this.maxFundable", this.maxFundable);
+    const min = this.bigNumberService.min([
+      this.targetClass.individualCap,
+      this.targetClass.classCap,
+      this.seed.userFundingTokenBalance,
+      this.maxFundable,
+    ]);
```
Comment (hiaux0) [inline src/seedSale/seedSale.ts:151]: these should all have the same decimals?
cc @giladt

File: src/seedSale/seedSale.html [hunk 2]
```diff
@@ -205,13 +221,13 @@
...
-              <div class="class-container__block__text--large">BACKER</div>
+              <div class="class-container__block__text--large">${targetClass.className}</div>
             </div>
             <div class="class-container__block">
               <div class="class-container__block__title">purchase limit</div>
               <div class="class-container__block__image-container">
                 <img class="class-container__block__image" src.to-view="seed.projectTokenInfo.logoURI" />
-                <div class="class-container__block__text"><formatted-number value.to-view="classCap | ethwei:seed.fundingTokenInfo.decimals" thousands-separated></formatted-number>/<formatted-number value.to-view="seed.cap | ethwei:seed.fundingTokenInfo.decimals" thousands-separated mantissa="0"></formatted-number></div>
+                <div class="class-container__block__text"><formatted-number value.to-view="0 | ethwei:seed.fundingTokenInfo.decimals" thousands-separated mantissa="0"></formatted-number>/<formatted-number value.to-view="targetClass.classCap | ethwei:seed.fundingTokenInfo.decimals" thousands-separated mantissa="0"></formatted-number></div>
```
[the remark this answers is not in the document]
Comment (hiaux0) [reply src/seedSale/seedSale.html:230]: it should show the amount the user has comtributed already (not indivCap), but yes, should not be "0"

File: src/seedSale/seedSale.ts (hunk 1 above)
Comment (Zitzak) [reply src/seedSale/seedSale.ts:151]: Not exactly sure which values these are, but in general you are right, all should have 18 decimals.

In regards to the USDC example and the different precisions, these should indeed be normalized to 18 decimals. This should be easily possible through EthersJS ‘parseUnits’  functionality

**Comment by hiaux0** [comment]
closing for #840

**Pull request closed without merging**